pkg/stack/types.go

```diff
@@ -136,6 +137,12 @@ type Stack struct {
 	Schedules   map[string]Schedule    `yaml:"schedules,omitempty"`
 	ApiDocs     map[string]*openapi3.T `yaml:"-"`
 	Apis        map[string]string      `yaml:"apis,omitempty"`
+	// TODO: Not currently supported by nitric.yaml configuration (but is technically definable using the proto model)
+	// We may want to decouple the definition from contracts at a later stage
+	// but re-using the contract here provides us a serializable entity with no
+	// repetition/redefinition
+	// NOTE: if we want to use the proto definition here we would need support for yaml parsing to use customisable tags
```

Comment:
Not sure if we would want to docouple this definition from the contract policies. I assumed these would end up being much the same anyway in the nitric.yaml.